src/schemas/gqlTypeDefs.ts

```diff
@@ -252,9 +252,11 @@ const typeDefs = gql`
 		fields: [ClinicalRecordField!]!
 	}
 
+	scalar ClinicalRecordValue
+
 	type ClinicalRecordField {
 		name: String!
-		value: String
+		value: ClinicalRecordValue
 	}
 
 	"""
```

Comment:
2 questions here:
- What is `ClinicalRecordValue`? Is there more of an implementation needed?
- For `ClinicalRecordField` I think "value" as "string" is correct, because it's for TSV